eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -389,13 +390,15 @@ object Commitments {
     }
   }
 
-  def receiveFee(commitments: Commitments, feeEstimator: FeeEstimator, feeTargets: FeeTargets, fee: UpdateFee, maxFeerateMismatch: Double): Try[Commitments] = {
+  def receiveFee(commitments: Commitments, feeEstimator: FeeEstimator, feeTargets: FeeTargets, fee: UpdateFee, maxFeerateMismatch: Double)(implicit log: LoggingAdapter): Try[Commitments] = {
     if (commitments.localParams.isFunder) {
       Failure(FundeeCannotSendUpdateFee(commitments.channelId))
     } else if (fee.feeratePerKw < fr.acinq.eclair.MinimumFeeratePerKw) {
       Failure(FeerateTooSmall(commitments.channelId, remoteFeeratePerKw = fee.feeratePerKw))
     } else {
+      Metrics.RemoteFeeratePerKw.withoutTags().record(fee.feeratePerKw)
```

Comment:
How will this make sense if we don't index by channel? (which we can't because of cardinality)

Reply:
Exactly, we can't index by channel so we just get a histogram of the overall mismatch.
The goal is to check how many channels we'd lose if we changed our tolerance to mismatch by N.
Then there's the added log line to check in details the mismatch of each channel.

Reply:
How would you do it differently?

Reply:
I missed that you were using a `histogram` vs a `gauge` for the local feerate. We'll be able to display the distribution of values, which alleviates the issue I was describing. Maybe we'll be able to group between various fee providers.